### DISCUSSION TITLE
Lesson 2 : StorageFactory.sol "Identifier not found or not unique"

### DISCUSSION THREAD

**Discussion by shucks18** [smartcontractkit/full-blockchain-solidity-course-py#807] [Q&A] [answered]
```Solidity
// SPDX-License-Identifier: MIT
pragma solidity ^0.6.0;

import "./SimpleStorage.sol";

contract StorageFactory{
    function createSimpleStorageContract() public {
        SimpleStorage simpleStorage = new SimpleStorage();
        
    }
     
}
```

// is there something wrong with my code?

**Comment by cromewar** [comment, marked as answer]
Hello @shucks18 is the name you used to create the file the same as the contract name StorageFactory?

**Reply by shucks18** [reply]
yeah it is :/

**Reply by cromewar** [reply]
Could you please share a screenshot of your remix, folder and that.

**Reply by shucks18** [reply]
![image](https://user-images.githubusercontent.com/81901871/149868016-bc525328-e56c-4d0f-a8a7-9a9cc1b86307.png)
so i realised that the problem was in the contract folder, mine was 'mycontract' - contracts/SimpleStorage instead of 'SimpleStorage' - contracts/SimpleStorge

**Reply by shucks18** [reply]
anyway i appreciate the help !!

**Comment by kellspell** [comment]
Hi guys good morning I still having the same problem here I did renamed the folder as suggest but the Error persists Help pls

**Reply by Magendra-ssn** [reply]
Hi, did you make sure that simplestorage contract also has the same name as the file name?, this is how i solved it

**Reply by cromewar** [reply]
Hello @kellspell please share a screenshot of your remix IDE
